h2o-r/h2o-package/pkgdown/_pkgdown.yml

```diff
@@ -122,6 +122,8 @@ reference:
       - h2o.floor
       - h2o.flow
       - h2o.gainsLift
+      - h2o.gains_lift
+      - h2o.gains_lift_plot
```

Comment:
I see that you introduced `h2o.plot_gains_lift` but don't see any `h2o.gains_lift_plot`.
Also, having both would be very confusing to say the least.

Reply:
Thank you for catching that. We renamed the function and I forgot to remove the old name from the `pkgdown`.